frontend/public/components/_autocomplete.scss

```diff
@@ -2,7 +2,6 @@
   background-color: #fff;
   z-index: 5;
   width: 100%;
-  max-width: 270px;
```

Comment:
Do we know why this style was here previously and what other pages this might affect?

Reply:
This was just added by bipul for the column filter component. I verified this doesn't change the other filters adversely